package.json

```diff
@@ -5,8 +5,8 @@
   "author": "QlikTech International AB",
   "license": "MIT",
   "scripts": {
-    "build": "webpack",
-    "build-watch": "webpack --config ./webpack.config-dev.js --watch",
+    "build": "concurrently \"webpack\" \"npm run test\" \"npm run lint\" \"npm run jsdoc\"",
```

Comment:
Just so that all checks are done whilst building (it means I'm lazy and I only want to type one command in).